### DISCUSSION TITLE
Filter editing bug [Feb 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5265] [Discord Threads]
> 🙋‍♂️ **sebigboss**

**Comment by o1lab** [comment]
> 🙋‍♂️ **sebigboss**  
#### Master Database is a PostgreSQL database and I can se the filters in the filters table.  
#  
> 🙋‍♂️ **sebigboss**  
#### I just downgraded my account to commenter and everything was fine  
#  
> 🙋‍♂️ **sebigboss**  
#### Seems to be a bug with the editor rights: an editor can "add" a filter to a view (and alter the view), but cannot reverse it. Creators can reverse it and viewers are not able to alter the view.  
#  
> 🙋‍♂️ **sebigboss**  
#### *viewers and commenters  
#  
> 🙋‍♂️ **sebigboss**  
#### Could be related to issue 4228?  
#  
> 🙋‍♂️ **kyleecodes**  
#### I see issue 4228 is closed.    
####    
#### We may need to submit another issue on GH   
#### https://github.com/nocodb/nocodb/issues/new?assignees=o1lab&labels=%7B%22Type%22%3D%3E%22Bug%22%7D&template=--bug-report.yaml&title=%F0%9F%90%9B+Bug%3A+   
####    
#### Thanks for reporting this!  
#  
> 🙋‍♂️ **sebigboss**  
#### 4228 was in 0.105.3, so I'll try upgrading before saying that did not fix the issue. 🙂  
#  
> 🙋‍♂️ **sebigboss**  
#### <@446452643221602304>  I can confirm that this bug is no longer present in 0.105.3 so I assume 4228 fixed it.  
#  
> 🙋‍♂️ **kyleecodes**  
#### Fantastic appreciate the update  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1080173922265337906)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
